ntbs-service/Pages/Notifications/Edit/_TravelDetailsPartial.cshtml

```diff
@@ -3,7 +3,11 @@
 @using ntbs_service.Models
 @using static NHSUK.FrontEndLibrary.TagHelpers.FormGroupType
 
-<validate-travel-or-visit model-type="travel" shouldvalidatefull="@Model.TravelDetails.ShouldValidateFull" inline-template>
+@{
+    var selectDropdownPlaceholder = "Select from list";
+    var fullValidation = Model.TravelDetails.ShouldValidateFull ? "True" : "False";
```

Comment:
Suddenly I had to do this, or shouldvalidatefull didn't get parsed - this definitely worked though before I started on the patient page argh